README.md

```diff
@@ -53,72 +98,30 @@ class Document
-*   Performance
-*   Reliability
-*   Monitoring
+## Why RedisMemo?
+
+1. [Database caching](https://github.com/chanzuckerberg/redis-memo/wiki/Database-caching): Quick review of why caching is important
+2. Challenges with application-level caching  
+    2.1 [Forgetting to invalidate the cache](https://github.com/chanzuckerberg/redis-memo/wiki/Auto-Invalidation-with-ActiveRecord#forgetting-to-invalidate-the-cache)  
+    2.2 [Cache invalidation during database transactions](https://github.com/chanzuckerberg/redis-memo/wiki/Cache-invalidation-during-database-transactions)  
+    2.3 [Cache invalidation could be slow and expensive](https://github.com/chanzuckerberg/redis-memo/wiki/Cache-invalidation-could-be-slow-and-expensive-with-Rails-low-level-caching)  
+    2.4 [Possible race conditions](https://github.com/chanzuckerberg/redis-memo/wiki/Possible-race-conditions-with-Rails-low-level-caching)  
+    2.5 [Cache inconsistency during deployments](https://github.com/chanzuckerberg/redis-memo/wiki/Ensure-consistency-during-deployments)  
+   
+3. How caching is easily done with RedisMemo  
+    3.1 [Performant and reliable cache invalidation](https://github.com/chanzuckerberg/redis-memo/wiki/Cache-invalidation-could-be-slow-and-expensive-with-Rails-low-level-caching#with-redismemo)  
+    3.2 [Auto-invalidation](https://github.com/chanzuckerberg/redis-memo/wiki/Auto-Invalidation-with-ActiveRecord)  
+    3.3 [Add caching without changing any call sites](https://github.com/chanzuckerberg/redis-memo/wiki/Add-caching-without-changing-any-call-sites)  
+    3.4 Add caching confidently    
+        &nbsp;&nbsp;&nbsp;&nbsp;3.4.1 [Avoid mistakes by pulling in existing dependencies](https://github.com/chanzuckerberg/redis-memo/wiki/Hierarchical-Caching#reuse-dependency)  
+        &nbsp;&nbsp;&nbsp;&nbsp;3.4.2 [Monitoring](https://github.com/chanzuckerberg/redis-memo/wiki/Monitoring)  
+        &nbsp;&nbsp;&nbsp;&nbsp;3.4.3 [Safely roll out changes](https://github.com/chanzuckerberg/redis-memo/wiki/Configure-RedisMemo#cache-sample-validation)  
+
+## Related Work
+We’re aware of [Shopify/identity_cache](https://github.com/Shopify/identity_cache), a gem that provides query caching with automatic cache invalidation; however, it is affected by most of the other issues we want to address when caching queries at the application-level. You can learn more about the other issues, in addition to the ones we are discussing here, when using the Rails low-level caching API or traditional caching technologies such as IdentityCache [here](https://github.com/chanzuckerberg/redis-memo/wiki).
+
+IdentityCache documented the motivation for providing separate cached query code paths in its [caveats](https://github.com/Shopify/identity_cache#caveats). With RedisMemo, caching is still a deliberate decision: there are models and computations that would not benefit from caching. When caching does make sense, RedisMemo makes caching easy and robust.
```

Comment:
```suggestion
IdentityCache documents the motivation for providing separate cached query code paths in its [caveats](https://github.com/Shopify/identity_cache#caveats). With RedisMemo, caching is still a deliberate decision: there are models and computations that would not benefit from caching. When caching does make sense, RedisMemo makes caching easy and robust.
```
I generally think this could use some rewording. A suggestion (feel free to ignore though): 

"IdentityCache is deliberately opt-in for all call sites that want to use caching. In comparison, RedisMemo is still deliberate in that clients should specify what computation and models should be cached. However, when caching does make sense, RedisMemo makes caching easy and robust by automatically using the cached code paths."

Reply:
Woo thanks! I like your version better :)